cmd/containers-storage/mount.go

```diff
@@ -51,12 +51,15 @@ func unmount(flags *mflag.FlagSet, action string, m storage.Store, args []string
 	mes := []mountPointError{}
 	errors := false
 	for _, arg := range args {
-		err := m.Unmount(arg)
+		mounted, err := m.Unmount(arg, force)
 		errText := ""
 		if err != nil {
 			errText = fmt.Sprintf("%v", err)
 			errors = true
 		}
+		if !mounted {
+			fmt.Printf("%s mountpoint unmounted\n", arg)
+		}
```

Comment:
Non-blocker: in principle it seems... off to treat `mounted` as useful if `err` is set.